### PR TITLE
Add support for hexnicks 3

### PR DESCRIPTION
**Pull request by MagnetiteMC**

### CODE REVIEW

File: src/main/java/me/dadus33/chatitem/playernamer/hook/HexNicksV3PlayerNamer.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import me.dadus33.chatitem.playernamer.IPlayerNamer;
+import net.kyori.adventure.text.serializer.bungeecord.BungeeComponentSerializer;
+import net.md_5.bungee.api.chat.BaseComponent;
+
+public class HexNicksV3PlayerNamer implements IPlayerNamer {
+
+    @Override
+    public BaseComponent[] getName(Player p) {
+        return BungeeComponentSerializer.get().serialize(HexNicks.api().getStoredNick(p).join());
```
Comment (Elikill58) [inline src/main/java/me/dadus33/chatitem/playernamer/hook/HexNicksV3PlayerNamer.java:14]: Seems to be the exact same line than the v2 with same imports.

File: src/main/java/me/dadus33/chatitem/playernamer/hook/HexNicksV2PlayerNamer.java [hunk 2]
```diff
@@ -7,10 +7,11 @@
 import net.kyori.adventure.text.serializer.bungeecord.BungeeComponentSerializer;
 import net.md_5.bungee.api.chat.BaseComponent;
 
-public class HexNicksPlayerNamer implements IPlayerNamer {
+public class HexNicksV2PlayerNamer implements IPlayerNamer {
 
 	@Override
 	public BaseComponent[] getName(Player p) {
+
```
Comment (Elikill58) [inline src/main/java/me/dadus33/chatitem/playernamer/hook/HexNicksV2PlayerNamer.java]: useless change xd

**Review by Elikill58 — CHANGES_REQUESTED** [review]
The behavior seems to be the same when it's with v2 or v3, I think it will not works. Can you make this changes? :)

File: src/main/java/me/dadus33/chatitem/playernamer/hook/HexNicksV3PlayerNamer.java (hunk 1 above)
Comment (MagnetiteMC) [reply src/main/java/me/dadus33/chatitem/playernamer/hook/HexNicksV3PlayerNamer.java:14]: Difference is HexNicks.api() vs Nicks.api()

**Review by Elikill58 — APPROVED** [review]

**Pull request merged**